rht-labs/omp-data-api
# Convert backup to MGOB

#### What does this PR do?
This PR converts the backup process to use MGOB (https://github.com/stefanprodan/mgob). This will currently use my fork of MGOB until we can upstream some of the required changes to run in OpenShift. This will allow us to have multiple backup schedules/locations with the added benefit of metrics.

#### How should this be tested?
Run the applier to deploy the app, db and backup agent. Provide it a short schedule so you can see the backup take place and then check your backup location for the backup.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @rht-labs/omp
